helpers.sh

```diff
@@ -16,19 +16,31 @@ readonly ANSI_RESET="\033[0m"
 readonly ANSI_CLEAR="\033[0K"
 
 print_info() {
-  printf "${ANSI_BOLD}${ANSI_BLUE}%s${ANSI_RESET}\n" "$1"
+  if [ "${config_color:-true}" == "true" ];
+  then printf "${ANSI_BOLD}${ANSI_BLUE}%s${ANSI_RESET}\n" "$1"
+  else echo "$1"
+  fi
 }
 
 print_notice() {
-  printf "${ANSI_BOLD}${ANSI_YELLOW}%s${ANSI_RESET}\n" "$1"
+  if [ "${config_color:-true}" == "true" ];
+  then printf "${ANSI_BOLD}${ANSI_YELLOW}%s${ANSI_RESET}\n" "$1"
+  else echo "$1"
+  fi
 }
 
 print_success() {
-  printf "${ANSI_BOLD}${ANSI_GREEN}%s${ANSI_RESET}\n" "$1"
+  if [ "${config_color:-true}" == "true" ];
+  then printf "${ANSI_BOLD}${ANSI_GREEN}%s${ANSI_RESET}\n" "$1"
+  else echo "$1"
+  fi
 }
 
 print_error() {
-  printf "${ANSI_BOLD}${ANSI_RED}%s${ANSI_RESET}\n" "$1" 1>&2
+  if [ "${config_color:-true}" == "true" ];
+  then printf "${ANSI_BOLD}${ANSI_RED}%s${ANSI_RESET}\n" "$1" 1>&2
+  else echo "$1"
```

Comment:
Is there a reason why this is not redirected to `stderr` or should we be add `1>&2` here?

Reply:
Good catch, i'll change that.
I think there are some places which do not use the helper functions which still output color codes, so this still requires work.

Reply:
Ok, let me know when you want me to have another look.